modular_skyrat/modules/admin/code/player_alerts/preferences.dm

```diff
@@ -0,0 +1,42 @@
+/datum/preferences
```

Comment:
Woah tiger, we have a preferences file.

Reply:
you aren't my preference, stop pref breaking me right now.

Reply:
You still gotta fix this @jjpark-kb